node/index.ts

```diff
@@ -30,5 +30,9 @@ export default new Service<IOClients>({
       ctx.response.status = 204
       ctx.response.body = 'My Account app is installed in this Store.'
     },
+    enabledMyOrders: (ctx: Context) => {
+      ctx.response.status = 204
+      ctx.response.body = 'My Account app is installed in this Store.'
```

Comment:
Did you mean to respond to both requests with the same message? 
```suggestion
      ctx.response.body = 'My Orders app is installed in this Store.'
```